eng/pipelines/templates/jobs/azuresdkpartnerdrops-to-nugetfeed.yml

```
@@ -61,53 +67,52 @@ jobs:
           AZCOPY_SPA_CLIENT_SECRET: $(azuresdkpartnerdrops-service-principal-key)
         condition: and(succeeded(), ne(variables['SkipCopyFromPartnerDrops'], 'true'))
 
-      - ${{ if ne(variables['SkipSigning'], 'true') }}:
+      - ${{ if eq(parameters.ShouldSign, 'true') }}:
```

I think since the type is explicit now, you can do a direct comparison for clarity/consistency, instead of relying on the devops type coercion from `'true'` to `true`.

```suggestion
      - ${{ if eq(parameters.ShouldSign, true) }}:
```